docs/content/docs/dev/table/sql/analyze.md

```diff
@@ -81,8 +81,8 @@ tableEnv.executeSql(
         " `id` BIGINT NOT NULl," +
         " `product` VARCHAR(32)," +
         " `amount` INT," +
-        " `sold_year` BIGINT", +
-        " `sold_month` BIGINT", +
+        " `sold_year` BIGINT," +
+        " `sold_month` BIGINT," +
```

Comment:
Please also check the Chinese document to see if there are any similar issues.

Reply:
Thanks for your reply. I will check Chinese document also.